src/components/variant/ui/drawer-window.tsx

```diff
@@ -0,0 +1,234 @@
+import { MouseEvent, ReactElement, useRef, useState } from 'react'
+import ScrollContainer from 'react-indiana-drag-scroll'
+import Checkbox from 'react-three-state-checkbox'
+import cn from 'classnames'
+import { get } from 'lodash'
+import { observer } from 'mobx-react-lite'
+import Tooltip from 'rc-tooltip'
+
+import { IGridLayout } from '@declarations'
+import { t } from '@i18n'
+import {
+  ICommonAspectDescriptor,
+  IPreAspectDescriptor,
+  ITableAspectDescriptor,
+  TRecCntResponse,
+} from '@service-providers/dataset-level/dataset-level.interface'
+
+const normClass = 'norm'
```

Comment:
Why not enum?

Reply:
This structure existed before me. In order not to increase the PR in size, I decided not to make minor changes